app/src/main/java/com/capstone/event_finder/network/EventViewModel.java

```diff
@@ -40,8 +48,57 @@ public void refreshEvents() {
         });
     }
 
-    public LiveData<List<Event>> eventInCache(String eventId) {
-        return eventRepository.eventInCache(eventId);
+    public LiveData<List<Event>> getBookmarks(ParseUser user) {
+        JsonArray allBookmarks = new JsonArray();
+        queryUserBookmarksFromParse(allBookmarks, user);
+        return bookmarkList;
+    }
+
+    private void tryRetrieveEventInCache(String eventId, JsonArray allBookmarks, List<Event> bookmarks) {
```

Comment:
💡 Nit: I'm not sure we need either this `allBookmarks` list nor this new `bookmarks` list.

We can probably just update the live data `bookmarkList` directly as we go.

Reply:
Check PR comment above